### PR TITLE
Eslintify panel

### PR DESCRIPTION
**Pull request by nojvek**
No change in `build` dir. Mostly syntactic sugar.

### CODE REVIEW

File: test/fixtures/attr-reflection-app.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
-import { Component, h } from '../../lib';
+import {Component, h} from '../../lib';
 
 export class AttrReflectionApp extends Component {
   get config() {
     return {
-      template: state => h('div', {class: {'attr-app': true}}, [
-        h('p', `Value of attribute wombats: ${this.getAttribute('wombats')}`),
+      // eslint-disable-next-line no-unused-vars
```
Comment (tdumitrescu) [inline test/fixtures/attr-reflection-app.js]: It's just a test fixture, might as well make it conform with `template: () => h(...)` (ditto for the other fixtures below)

File: test/server/dom-patcher.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+/* eslint-env mocha */
+/* eslint-disable no-unused-expressions */
```
[the remark this answers is not in the document]
Comment (nojvek) [reply test/server/dom-patcher.js]: `expect(domPatcher.el).to.be.ok;`

it isn't ok with `to.be.ok`

**Comment by nojvek** [comment]
Not creating a new release since there is no change in build.

**Comment by tdumitrescu** [comment]
Did you actually check the build files against the current release? I don't see how they could have stayed the same, with the removal of the lodash import.

**Comment by nojvek** [comment]
you are right. I was assuming a git status change would have shown a change in build. My bad, publishing a patched version

**Comment by nojvek** [comment]
+ panel@1.3.2

**Pull request merged**